pythnet/pythnet_sdk/src/messages.rs

```diff
@@ -0,0 +1,330 @@
+use {
+    crate::wire::DeserializerError,
+    byteorder::{
+        BigEndian,
+        ReadBytesExt,
+    },
+    std::io::{
+        Cursor,
+        Read,
+    },
+};
+
+
+/// Message format for sending data to other chains via the accumulator program
+/// When serialized, each message starts with a unique 1-byte discriminator, followed by the
```

Comment:
Minor nitpick, maybe clarify this is with the PythNet serialization format, as this won't be true for any other serde based serializer like JSON.